Source/Core/Core/HW/Memmap.cpp

```diff
@@ -102,17 +103,60 @@ bool IsInitialized()
 }
 
 
-// We don't declare the IO region in here since its handled by other means.
+// Dolphin allocates memory to represent four regions:
+// - 24MB RAM, available on Gamecube and Wii
+// - 64MB "EXRAM", RAM only available on Wii
+// - 64MB FakeVMem, allocated when MMU support is turned off. This is used
```

Comment:
Err, that's a typo; we actually allocate 32MB.

As to why, I have no idea; it's been done that way since before I ever looked at the code in question.